app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False
+                 ) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def die(self) -> None:
+        if self.health <= 0 and self in Animal.alive:
+            Animal.alive.remove(self)
+
+    def __repr__(self) -> str:
+        return "{{Name: {}, Health: {}, Hidden: {}}}".format(
+            self.name, self.health, self.hidden
+        )
+
+    @classmethod
+    def __str__(cls) -> str:
```

Comment:
The `__str__` method should not be a class method. It is typically an instance method that returns a string representation of an instance. Consider removing the `@classmethod` decorator and adjusting the method to return a string representation of an instance, not the class.